terracumber_config/tf_files/Uyuni-Master-build-validation-NUE.tf

```diff
@@ -638,8 +638,8 @@ module "sles15sp5s390-minion" {
 }
 
 module "salt-migration-minion" {
-  source = "./modules/minion"
-  base_configuration = module.base.configuration
+  source             = "./modules/minion"
+  base_configuration = module.base_core.configuration
```

Comment:
I would better use the same base for all the terraform files, `base_new_sle` is fine.
But please, avoid using `base_core`, having this setup, it will deploy this VM together with the server and core components, and we want to keep core components as much isolated as possible.

Reply:
what you say is true for PRV, not for NUE, where we have only one hypervisor